tests/cli-rustup.rs

```diff
@@ -23,7 +23,7 @@ macro_rules! for_host_and_home {
     };
 }
 
-pub fn setup(f: &dyn Fn(&Config)) {
+fn setup(f: &dyn Fn(&Config)) {
```

Comment:
While it doesn't need to be `pub` -- this is a wart on the change, could you please remove this bit of the diff?